### PR TITLE
Disable a list of monitor tests for resource watch

### PR DESCRIPTION
**Pull request by xueqzhan**
[TRT-1269](https://issues.redhat.com/browse/TRT-1269)

### CODE REVIEW

File: ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh [hunk 1]
```diff
@@ -48,5 +48,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-openshift-tests run-monitor --artifact-dir $STORE_PATH > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
+DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
+openshift-tests run-monitor --artifact-dir $STORE_PATH --disable-monitor=${DISABLED_MONITOR_TESTS} > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
```
Comment (dgoodwin) [inline ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh:52]: Is it possible to --enable-monitor to only run what we want? so we don't have to maintain this over time? Or do we need too many monitor tests.

Comment (xueqzhan) [reply ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh:52]: We will have a bigger list to include. We have total of 36 tests, only 8 to be excluded. 

Here is the list:

additional-events-collector, alert-summary-serializer, apiserver-availability, apiserver-new-disruption-invariant, audit-log-analyzer, cluster-info-serializer, clusteroperator-collector, disruption-summary-serializer, e2e-test-analyzer, etcd-log-analyzer, event-collector, external-service-availability, graceful-shutdown-analyzer, image-registry-availability, ingress-availability, interval-serializer, known-image-checker, kubelet-log-collector, legacy-authentication-invariants, legacy-cvo-invariants, legacy-etcd-invariants, legacy-kube-apiserver-invariants, legacy-networking-invariants, legacy-node-invariants, legacy-storage-invariants, legacy-test-framework-invariants, node-lifecycle, node-state-analyzer, operator-state-analyzer, pathological-event-analyzer, pod-lifecycle, pod-network-avalibility, service-type-load-balancer-availability, timeline-serializer, tracked-resources-serializer, upload-to-loki-serializer

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-sdn-upgrade

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-sdn-upgrade

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-ovn-serial

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-ovn-serial

**Comment by dgoodwin** [comment]
Branch needs a rebase but this looks good to me given the explanation above.

File: ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh [hunk 2]
```diff
@@ -61,5 +61,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-openshift-tests run-monitor ${MONITOR_ARGS:-} --artifact-dir $STORE_PATH > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
+DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
```
Comment (vrutkovs) [inline ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh:64]: Please add `incluster-disruption-serializer` as well, https://github.com/openshift/origin/pull/28277 would add that

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-ovn-serial

**Pull request merged**